node/tests/SharedTests.ts

```diff
@@ -873,6 +873,60 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `lset with nonexisting key`,
```

Comment:
I see that some of the other tests mention that they are testing with a nonexisting key, but for those tests they also seem to have other tests. Since we're testing all lset functionality in this one place instead of the nonexisting key specifically, can we make the test name more general? The other tests seem to have this format:
```suggestion
        `lset test_%p`,
```